src/version-store.js

```diff
@@ -0,0 +1,7 @@
+// Index of each item is equivalent to the major of zapier-platform-core version.
+// The next item will be for zapier-platform-core version of major 3.
+module.exports = [
+  {nodeVersion: '>=4.3.2', npmVersion: '>=2.14.12'},
+  {nodeVersion: '>=4.3.2', npmVersion: '>=2.14.12'},
+  {nodeVersion: '>=6.10.2', npmVersion: '>=3.10.10'}
```

Comment:
If we're requiring exact node versions, is it worth proving those instead of `>=`? npm has some wiggle room, so it's less of a big deal.